src/OpenTelemetry.Api/Trace/SpanAttributeConstants.cs

```diff
@@ -19,7 +19,7 @@ namespace OpenTelemetry.Trace
     /// <summary>
     /// Defines well-known span attribute keys.
     /// </summary>
-    public static class SpanAttributeConstants
+    internal static class SpanAttributeConstants
```

Comment:
i am wondering why is the instrumentation populating those tags which are not part of semantic convention? we can eliminate that, and then eliminate the need for this at all.

(Lets address it separately, Okay to merge this)